src/flow.ts

```diff
@@ -224,7 +224,9 @@ export class Flow {
     flow.returnType = parentFunction.signature.returnType;
     flow.contextualTypeArguments = parentFunction.contextualTypeArguments;
     flow.localFlags = [];
-    flow.fieldFlags = new Map();
+    if (parentFunction.is(CommonFlags.CONSTRUCTOR)) {
```

Comment:
Should be `actualFunction`. Where inlining is performed, for example when inlining a constructor into another function, `parentFunction` is the function being inlined into (hosting all the locals etc), while `actualFunction` is the one becoming inlined (more of an indicator, doesn't hold much state).

Reply:
Oh I was just checking this case right now and was wondering what was the difference. I'll change it.

Reply:
Ah also makes sense and that's why we check against `actualFunction` everytime we want to know if we are in the constructor.